### PR TITLE
Fix jQuery version

### PR DESCRIPTION
**Pull request by vsn4ik**
https://github.com/npm/node-semver#hyphen-ranges-xyz---abc: `3.0.0 - 4.0.0` := `>=3.0.0 <=4.0.0`
https://github.com/npm/node-semver#caret-ranges-123-025-004: `^3.0.0` := `>=3.0.0 <4.0.0`

For test - https://semver.npmjs.com/:
- pick a package - `jquery`
- enter a range - `2.0.0 - 3.0.0`
- enter a range - `^2.0.0`

### CODE REVIEW

**Comment by Johann-S** [comment]
Yes but the range we want is : `3.0.0 - 4.0.0`

**Comment by XhmikosR** [comment]
I think `^3.0.0` is fine because if `4.0.0` it won't be installed anyway based on semver. The only thing is if we want us to be explicit here, nothing more.

About jspm I can't comment since I have never used it.

**Comment by vsn4ik** [comment]
@Johann-S no, `if (version[0] < min || version[0] >= max) {` this is a test for incorrectness (inversion `if (version[0] >= min && version[0] < max`).

**Comment by Johann-S** [comment]
Yes I agree but that's the limit `min` and `max` 😉 but that's ok for me and I never used jspm me too 😟

**Comment by vsn4ik** [comment]
Fix conflict in `bower.json`

File: bower.json [hunk 1]
```diff
@@ -1 +1 @@
-@$% Not a Bower package. Use a better package management tool instead. ^&*
\ No newline at end of file
+@$% Not a Bower package. Use a better package management tool instead. ^&*
```
Comment (XhmikosR) [inline bower.json]: Can you skip touching this file altogether?

Comment (vsn4ik) [reply bower.json]: Done

**Review by XhmikosR — APPROVED** [review]

**Comment by XhmikosR** [comment]
@Johann-S: feel free to merge. From my understanding using `^` has the same effect.

**Pull request merged**